src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -4999,6 +4999,23 @@ This only works well if you're using a single control in the form field.
       "optional": true,
       "type": "string",
     },
+    Object {
+      "description": "An object containing all the necessary localized strings required by the component.",
```

Comment:
"An object containing all localized strings required by the component."

Reply:
This is automatically generated by [this line of code.](https://github.com/cloudscape-design/components/pull/202/files/42aaff509366d7c123fe49fe66245198863f3454#diff-f94eadc24807d031eff51913e08cf8e23843e51183872ac76b0069e4b66561f8R37)